testproject/testproject/settings.py

```diff
@@ -146,3 +141,6 @@
 }
 
 WIKI_ANONYMOUS_WRITE = True
+
+# We disable this in edx-platform lms/envs/common.py, so disabling for test projet also.
+WIKI_USE_BOOTSTRAP_SELECT_WIDGET = False
```

Comment:
With the 1.11 upgrade, I was having trouble with a template used for this widget. Given that we have this disabled in edx-platform (and actually use our own templates there anyway), I didn't figure it was worth spending more time on.